notebooks/dataset_tutorial/sensorium/training/trainers.py

```diff
@@ -40,19 +36,8 @@ def standard_trainer(
     lr_decay_factor=0.3,
     min_lr=0.0001,
     cb=None,
-    use_wandb=True,
-    wandb_project="Rotation_test",
-    wandb_entity="ninasophie-nellen-g-ttingen-university",
```

Comment:
I see that its deleted, but as an advice - better not put real project names, etc to git
And NEVER put any keys (like wandb authorisation key to git)